notebooks/notebooks/iclr2022/winrate-predictions.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.            np.clip(smoothed_victim_bce_loss + smoothed_victim_bce_loss_std, a_min=0, a_max=None),
</pre><p>nit: I'm wondering whether we want to be shading in the loss +- std dev or whether we should be shading in a 95% confidence interval around the mean loss (i.e., loss +- 1.96 std dev or whatever it is)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Originally I did intend to use stddev to just show the overall dispersion of the data, not our confidence about the mean, but maybe that's confusing idk